openedx/core/djangoapps/heartbeat/views.py

```diff
@@ -8,6 +8,10 @@
 from util.json_request import JsonResponse
 from xmodule.exceptions import HeartbeatFailure
 from xmodule.modulestore.django import modulestore
+from dogapi import dog_stats_api
```

Comment:
Both `dog_stats_api` and `JsonResponse` were already imported.

In general, the imports in this module can stand to be cleaned up, lots of unneeded imports.